integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -215,8 +215,9 @@ public void testRestartOperatorAndVerifyScaling() {
   @Test
   @DisplayName("Restart operator when the domain is rolling after the admin credentials are changed")
   public void testOperatorRestartWhenPodRoll() {
-    verifyMiiUpdateWebLogicCredential(domainNamespace, domainUid, adminServerPodName,
+    verifyUpdateWebLogicCredential(domainNamespace, domainUid, adminServerPodName,
         managedServerPrefix, replicaCount);
+    restartOperatorAndVerify();
```

Comment:
Ok, since the added util method is for entire domain restart I reverted the change. Only minor change for patchDomainWithNewSecretAndVerify.
Thanks Vanaja.